examples/with-portals/components/Portal.js

```diff
@@ -2,15 +2,21 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 export class Portal extends React.Component {
+  constructor (props) {
+    super(props)
+    this.elem = document.createElement('div')
```

Comment:
This would break with SSR because constuctor() is executed on SSR.

The correct solution would be to call `this.forceUpdate()` in `componentDidMount` after setting `this.element`, so that the React component is forced to re-render.

Reply:
thanks, updated as u said.. btw, I'm curious, if we still use this approach and add if (process.browser), this approach will not break?